package.json

```diff
@@ -38,12 +38,14 @@
     "filesize": "^8.0.6",
     "formik": "^2.2.9",
     "gray-matter": "^4.0.3",
+    "i": "^0.3.7",
     "is-url-superb": "^6.1.0",
     "js-cookie": "^3.0.1",
     "jwt-decode": "^3.1.2",
     "lodash.debounce": "^4.0.8",
     "lodash.omit": "^4.5.0",
     "next": "^12.1.0",
+    "npm": "^8.5.1",
```

Comment:
do we really need npm install on the project?

Reply:
i have no idea how that got there